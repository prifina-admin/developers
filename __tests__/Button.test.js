import React, {createRef} from "react";
import renderer from "react-test-renderer";
import Button from "../src/Button";
import ThemeProvider from "../src/theme/ThemeProvider";

test("Button renders correctly", () => {
    const tree = renderer.create(<ThemeProvider>
        <Button>Something</Button>
        </ThemeProvider>).toJSON();
    expect(tree).toMatchSnapshot();
})
