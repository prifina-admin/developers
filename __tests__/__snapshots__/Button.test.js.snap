// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<div
  className="ThemeProvider__Base-sc-1rk52n4-0 emGfPA"
>
  <button
    className="Button__ButtonElement-sc-69esed-2 gZHanz"
    size="md"
  >
    Something
  </button>
</div>
`;
